### ISSUE TITLE
Fit Info Panel does not refresh on swapping tabs

### ISSUE THREAD

**Issue by jacobmatthewjansen** [pyfa-org/Pyfa#1551]
### Expected behavior:
When swapping from one fitting tab to another, I expect the fitting info tab to refresh and display the fit info for the newly selected tab/fit.  

### Actual behavior:
When swapping between fits by clicking on another tab, the fitting info panel 

### Detailed steps to reproduce:
Here is a Gif of me reproducing the bug:
https://i.imgur.com/iKEBZxY.gifv

1. Open up two fits in Pyfa
2. Select the first by clicking on its tab and ensure the fit info panel is correct
3. Swap to the second fit by clicking on its tab
4. Notice that the info panel **DOES** refresh with the new fit
5. Swap back to previous fit
6. Notice the info panel **DOES NOT**  refresh with the new fit

### Fits involved in EFT format (Edit > To Clipboard > EFT):
```
[Vexor, pyfa test 1: DDA]

Drone Damage Amplifier II
Drone Damage Amplifier II
Drone Damage Amplifier II
Drone Damage Amplifier II
Drone Damage Amplifier II

[Empty Med slot]
[Empty Med slot]
[Empty Med slot]
[Empty Med slot]

[Empty High slot]
[Empty High slot]
[Empty High slot]
[Empty High slot]

[Empty Rig slot]
[Empty Rig slot]
[Empty Rig slot]


Ogre II x5
```

```
[Vexor, pyfa test 2: Empty]

[Empty Low slot]
[Empty Low slot]
[Empty Low slot]
[Empty Low slot]
[Empty Low slot]

[Empty Med slot]
[Empty Med slot]
[Empty Med slot]
[Empty Med slot]

[Empty High slot]
[Empty High slot]
[Empty High slot]
[Empty High slot]

[Empty Rig slot]
[Empty Rig slot]
[Empty Rig slot]
```

### Release or development git branch? Please note the release version or commit hash:
v2.0.0

### Operating system and version (eg: Windows 10, OS X 10.9, OS X 10.11, Ubuntu 16.10):
Windows 10

### Other relevant information:
N/A

**Comment by blitzmann** [comment]
I have a few reports on this, but I cannot for the life of me reproduce it. The fit always seems to refresh correctly.

Are you able to consistently reproduce this issue with the same fits? Am I correct in noticing that once you turn a module on / off, it recalculates correctly?

@Neugeniko @fsufitch @minlexx I know you guys are frequently around, so I would like a second set of eyes, can any of you reproduce or have noticed this issue?

**Comment by blitzmann** [comment]
Ah, btw, this was also addressed in #1551 and has been fixed. I was finally able to reproduce the issue. it should be fixed in the new 2.0.2 release. :)
